packages/core/src/utils/spanUtils.ts

```diff
@@ -31,20 +31,18 @@ export const TRACE_FLAG_SAMPLED = 0x1;
 
 /**
  * Convert a span to a trace context, which can be sent as the `trace` context in an event.
+ * By default, this will only include trace_id, span_id & parent_span_id.
+ * If `includeAllData` is true, it will also include data, op, status & origin.
  */
-export function spanToTraceContext(span: Span): TraceContext {
+export function spanToTraceContext(span: Span, includeAllData = false): TraceContext {
   const { spanId: span_id, traceId: trace_id } = span.spanContext();
   const { data, op, parent_span_id, status, origin } = spanToJSON(span);
 
-  return dropUndefinedKeys({
-    data,
-    op,
-    parent_span_id,
-    span_id,
-    status,
-    trace_id,
-    origin,
-  });
+  const reqFields = { parent_span_id, span_id, trace_id };
+  // These are only included if `includeAllData` is true
+  const optFields = { data, op, status, origin };
+
+  return dropUndefinedKeys(includeAllData ? { ...reqFields, ...optFields } : reqFields);
 }
```

Comment:
I would be heavily in favor of creating a separate function instead of passing a flag.

ie
- spanToTransactionTraceContext
- spanToTraceContext

or similar

Reply:
Reason being "includeAllData" is super undescriptive. What is _not_ all data? Why would I use that?

Reply:
sounds good to me! will update this.